plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -205,18 +203,6 @@ func (l *persistentVolumeLabel) findVolumeLabels(volume *api.PersistentVolume) (
 			return nil, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err)
 		}
 		return labels, nil
-	case volume.Spec.AzureDisk != nil:
```

Comment:
With this PR and https://github.com/kubernetes/kubernetes/pull/124519, I suggest some removing the entire admission plugin. It has been deprecated for many releases and we published an external alternative https://github.com/kubernetes-sigs/cloud-pv-admission-labeler

Reply:
I did not know about https://github.com/kubernetes-sigs/cloud-pv-admission-labeler. I agree that removing the admission is the best way forward now.